Library/Formula/docbook.rb

```diff
@@ -45,7 +45,14 @@ class Docbook < Formula
 
   def install
     (etc/"xml").mkpath
-    system "xmlcatalog", "--noout", "--create", "#{etc}/xml/catalog"
+
+    # only create catalog file if it doesn't exist already to avoid content added
+    # by other formulas to be removed
+    unless File.file?("#{etc}/xml/catalog")
```

Comment:
We can't relay on this. Especially for bottles. If this is really needed, it should be put in `post_install`.